docs/installation.rst

```diff
@@ -1,12 +1,16 @@
 Installation
 ============
 
-Channels is available on PyPI - to install it, just run::
+Channels is available on PyPI - to install it, just run:
 
-    pip install -U channels
+.. code-block:: sh
+
+    python -m pip install -U channels
```

Comment:
I'm so _Meh_ about this. `pip install...` has worked fine for years. 🙂



There was a matching one in `contributing.txt`. (I'm not suggesting you update that too. 🙂)